pkg/collector/metrics.go

```diff
@@ -0,0 +1,73 @@
+package collector
+
+import (
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/kyma-project/kyma-metrics-collector/pkg/runtime"
+)
+
+const (
+	namespace          = "kmc"
+	subsystem          = "collector"
+	shootNameLabel     = "shoot_name"
+	instanceIdLabel    = "instance_id"
+	runtimeIdLabel     = "runtime_id"
+	subAccountLabel    = "sub_account_id"
+	globalAccountLabel = "global_account_id"
+	successLabel       = "success"
+	resourceNameLabel  = "resource_name"
+	backendNameLabel   = "backend_name"
+	EDPBackendName     = "edp"
```

Comment:
lets reuse these constants also for the traces

Reply:
We have dedicated constants now in the `otel` package. So, the `SpanAttributes` helper func can be used by both collector and scanners